namespaces/kubernetes.yml

```diff
@@ -57,6 +68,32 @@ namespace:
       Each label name is a subfield of labels field.
       Each label name is de-dotted: dots in the name are replaced with
       underscores.
+    fields:
+      - name: deployment
+        type: string
+        example: logging-kibana-3
+        description: >
+          The deployment associated with this Kubernetes object
+      - name: deploymentconfig
+        type: string
+        example: logging-kibana
+        description: >
+          The deploymentconfig associated with this Kubernetes object
+        fields:
+          - name: raw
```

Comment:
I need to have a field in the index pattern file like this:
```
{\"count\": 0, \"name\": \"kubernetes.labels.deploymentconfig.raw\", \"indexed\": true, \"analyzed\": false, \"doc_values\": true, \"type\": \"string\", \"scripted\": false}
```
I'm not sure how to do that unless I explicitly define a `raw` subfield of `kubernetes.labels.deploymentconfig`